Failed to execute CTest

## System (系统)

```
NAME="Ubuntu"
VERSION="18.04.5 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.5 LTS"
VERSION_ID="18.04"
```

## Install (安装)

```
spack install --overwrite --keep-stage --no-checksum -y cgal@6.0
```

## Step to reproduce (步骤)

```
cd cgal@6.0
rm -rf build;mkdir build;cd build
source ../../spack-build-env.txt
cmake -DBUILD_TESTING=ON -DWITH_examples=ON -DWITH_tests=ON ..
ctest
```

## Error (报错)

```
The following tests FAILED:
        1956 - compilation of  edge_collapse_envelope (Failed)
        1957 - execution   of  edge_collapse_envelope (Not Run)
        1960 - compilation of  edge_collapse_constrain_sharp_edges (Failed)
        1961 - execution   of  edge_collapse_constrain_sharp_edges (Not Run)
        1962 - compilation of  edge_collapse_constrained_border_polyhedron (Failed)
        1963 - execution   of  edge_collapse_constrained_border_polyhedron (Not Run)
        1964 - compilation of  edge_collapse_enriched_polyhedron (Failed)
        1965 - execution   of  edge_collapse_enriched_polyhedron (Not Run)
        1966 - compilation of  edge_collapse_polyhedron (Failed)
        1967 - execution   of  edge_collapse_polyhedron (Not Run)
        1968 - compilation of  edge_collapse_surface_mesh (Failed)
        1969 - execution   of  edge_collapse_surface_mesh (Not Run)
        1970 - compilation of  edge_collapse_linear_cell_complex (Failed)
        1971 - execution   of  edge_collapse_linear_cell_complex (Not Run)
        1972 - compilation of  edge_collapse_constrained_border_surface_mesh (Failed)
        1973 - execution   of  edge_collapse_constrained_border_surface_mesh (Not Run)
        1974 - compilation of  edge_collapse_all_short_edges (Failed)
        1975 - execution   of  edge_collapse_all_short_edges (Not Run)
        1976 - compilation of  edge_collapse_bounded_normal_change (Failed)
        1977 - execution   of  edge_collapse_bounded_normal_change (Not Run)
        1978 - compilation of  edge_collapse_visitor_surface_mesh (Failed)
        1979 - execution   of  edge_collapse_visitor_surface_mesh (Not Run)
        1980 - compilation of  edge_collapse_garland_heckbert (Failed)
        1981 - execution   of  edge_collapse_garland_heckbert (Not Run)

Building CXX object examples/Surface_mesh_simplification/CMakeFiles/edge_collapse_linear_cell_complex.dir/edge_collapse_linear_cell_complex.cpp.o
In file included from /home/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-6.0-xqznqg3c45uwesjsy7n632fv2vzyxkdo/include/CGAL/Surface_mesh_simplification/Policies/Edge_collapse
/LindstromTurk_cost.h:17:0,
                 from /home/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-6.0-xqznqg3c45uwesjsy7n632fv2vzyxkdo/include/CGAL/Surface_mesh_simplification/Policies/Edge_collapse
/LindstromTurk.h:16,
                 from /home/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-6.0-xqznqg3c45uwesjsy7n632fv2vzyxkdo/include/CGAL/Surface_mesh_simplification/edge_collapse.h:22,
                 from /home/stage/root/spack-stage-cgal-6.0-xqznqg3c45uwesjsy7n632fv2vzyxkdo/spack-src/examples/Surface_mesh_simplification/edge_collapse_linear_cell_complex.cpp:4:
/home/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-6.0-xqznqg3c45uwesjsy7n632fv2vzyxkdo/include/CGAL/Surface_mesh_simplification/Policies/Edge_collapse/internal/Lindstrom_Tu
rk_core.h:204:32: error: macro "minor" passed 4 arguments, but takes just 1
     FT x = minor(ay, by, az, bz);
                                ^
/home/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-6.0-xqznqg3c45uwesjsy7n632fv2vzyxkdo/include/CGAL/Surface_mesh_simplification/Policies/Edge_collapse/internal/Lindstrom_Tu
rk_core.h:205:32: error: macro "minor" passed 4 arguments, but takes just 1
     FT y = minor(az, bz, ax, bx);
                                ^
/home/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-6.0-xqznqg3c45uwesjsy7n632fv2vzyxkdo/include/CGAL/Surface_mesh_simplification/Policies/Edge_collapse/internal/Lindstrom_Tu
rk_core.h:206:32: error: macro "minor" passed 4 arguments, but takes just 1
     FT z = minor(ax, bx, ay, by);
                                ^
In file included from /home/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/boost-1.76.0-fji3lbyd2qxnhvnmudqxc4um3aahm3zg/include/boost/graph/adjacency_iterator.hpp:13:0,
                 from /home/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/boost-1.76.0-fji3lbyd2qxnhvnmudqxc4um3aahm3zg/include/boost/graph/detail/adjacency_list.hpp:37,
                 from /home/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/boost-1.76.0-fji3lbyd2qxnhvnmudqxc4um3aahm3zg/include/boost/graph/adjacency_list.hpp:255,
                 from /home/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-6.0-xqznqg3c45uwesjsy7n632fv2vzyxkdo/include/CGAL/Surface_mesh_simplification/internal/Common.h:27,
                 from /home/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/cgal-6.0-xqznqg3c45uwesjsy7n632fv2vzyxkdo/include/CGAL/Surface_mesh_simplification/edge_collapse.h:20,
                 from /home/stage/root/spack-stage-cgal-6.0-xqznqg3c45uwesjsy7n632fv2vzyxkdo/spack-src/examples/Surface_mesh_simplification/edge_collapse_linear_cell_complex.cpp:4:
/home/spack/opt/spack/linux-ubuntu18.04-aarch64/gcc-7.5.0/boost-1.76.0-fji3lbyd2qxnhvnmudqxc4um3aahm3zg/include/boost/detail/iterator.hpp:13:37: note: #pragma message: This header is de
precated. Use <iterator> instead.
 BOOST_HEADER_DEPRECATED("<iterator>")
                                     ^
examples/Surface_mesh_simplification/CMakeFiles/edge_collapse_linear_cell_complex.dir/build.make:81: recipe for target 'examples/Surface_mesh_simplification/CMakeFiles/edge_collapse_lin
ear_cell_complex.dir/edge_collapse_linear_cell_complex.cpp.o' failed
make[3]: *** [examples/Surface_mesh_simplification/CMakeFiles/edge_collapse_linear_cell_complex.dir/edge_collapse_linear_cell_complex.cpp.o] Error 1
CMakeFiles/Makefile2:29299: recipe for target 'examples/Surface_mesh_simplification/CMakeFiles/edge_collapse_linear_cell_complex.dir/all' failed
make[2]: *** [examples/Surface_mesh_simplification/CMakeFiles/edge_collapse_linear_cell_complex.dir/all] Error 2
CMakeFiles/Makefile2:29306: recipe for target 'examples/Surface_mesh_simplification/CMakeFiles/edge_collapse_linear_cell_complex.dir/rule' failed
make[1]: *** [examples/Surface_mesh_simplification/CMakeFiles/edge_collapse_linear_cell_complex.dir/rule] Error 2
Makefile:11660: recipe for target 'edge_collapse_linear_cell_complex' failed
make: *** [edge_collapse_linear_cell_complex] Error 2
<end of output>
Test time =  17.12 sec
```

--- a/Surface_mesh_simplification/include/CGAL/Surface_mesh_simplification/Policies/Edge_collapse/internal/Lindstrom_Turk_core.h
+++ b/Surface_mesh_simplification/include/CGAL/Surface_mesh_simplification/Policies/Edge_collapse/internal/Lindstrom_Turk_core.h
@@ -190,7 +190,7 @@ private :
     const FT ax=a.x(), ay=a.y(), az=a.z();
     const FT bx=b.x(), by=b.y(), bz=b.z();
 
-    auto minor = [](double ai, double bi, double aj, double bj)
+    auto compute_minor = [](double ai, double bi, double aj, double bj)
     {
       // The main idea is that we expect ai and bi (and aj and bj) to have roughly the same magnitude
       // since this function is used to compute the cross product of two vectors that are defined
@@ -201,9 +201,9 @@ private :
     };
 
     // ay*
-    FT x = minor(ay, by, az, bz);
-    FT y = minor(az, bz, ax, bx);
-    FT z = minor(ax, bx, ay, by);
+    FT x = compute_minor(ay, by, az, bz);
+    FT y = compute_minor(az, bz, ax, bx);
+    FT z = compute_minor(ax, bx, ay, by);
 
     return Vector(x, y, z);
   }

